blog/2023-06-01-danke/index.md

```diff
@@ -0,0 +1,40 @@
+---
+page-title: 'The Corona-Warn-App team says &#34;Thank You!&#34;'
+page-description: 'The Corona-Warn-App team says &#34;Thank You!&#34;'
+page-name: thank-you
+page-name_de: thank-you
+author: CWA-Team, 10:00 am
+layout: blog
+---
+
+Today marks the end of more than three years of development work on one of the most important digital projects in the pandemic: the Corona Warning App. Since its launch on June 16, 2020, the CWA has been downloaded more than 48 million times. More than nine million alerts have been triggered since then, and many millions of people have been notified of risk encounters with red and green tiles on their smartphones.

```

Comment:
This website has consistently referred to CWA as "Corona-Warn-App" not "Corona Warning App" in English. The same term as is used in the English version of the app stores.

Later on in the text "Corona-Warn-App" is used.

So I suggest to change it here too.

Reply:
many thanks!